test/Unit/PowerShell/PowerShellManagerTests.cs

```diff
@@ -14,44 +14,76 @@
 
 namespace Microsoft.Azure.Functions.PowerShellWorker.Test
 {
-    public class PowerShellManagerTests
+    internal class TestUtils
     {
-        private const string TestInputBindingName = "req";
-        private const string TestOutputBindingName = "res";
-        private const string TestStringData = "Foo";
+        internal const string TestInputBindingName = "req";
+        internal const string TestOutputBindingName = "res";
 
-        private readonly string _functionDirectory;
-        private readonly ConsoleLogger _testLogger;
-        private readonly PowerShellManager _testManager;
-        private readonly List<ParameterBinding> _testInputData;
-        private readonly RpcFunctionMetadata _rpcFunctionMetadata;
-        private readonly FunctionLoadRequest _functionLoadRequest;
+        internal static readonly string FunctionDirectory;
+        internal static readonly RpcFunctionMetadata RpcFunctionMetadata;
+        internal static readonly FunctionLoadRequest FunctionLoadRequest;
 
-        public PowerShellManagerTests()
+        static TestUtils()
         {
-            _functionDirectory = Path.Join(AppDomain.CurrentDomain.BaseDirectory, "TestScripts", "PowerShell");
-            _rpcFunctionMetadata = new RpcFunctionMetadata()
+            FunctionDirectory = Path.Join(AppDomain.CurrentDomain.BaseDirectory, "TestScripts", "PowerShell");
+            RpcFunctionMetadata = new RpcFunctionMetadata()
             {
                 Name = "TestFuncApp",
-                Directory = _functionDirectory,
+                Directory = FunctionDirectory,
                 Bindings =
                 {
                     { TestInputBindingName , new BindingInfo { Direction = BindingInfo.Types.Direction.In, Type = "httpTrigger" } },
                     { TestOutputBindingName, new BindingInfo { Direction = BindingInfo.Types.Direction.Out, Type = "http" } }
                 }
             };
-            _functionLoadRequest = new FunctionLoadRequest {FunctionId = "FunctionId", Metadata = _rpcFunctionMetadata};
-            FunctionLoader.SetupWellKnownPaths(_functionLoadRequest);
 
+            FunctionLoadRequest = new FunctionLoadRequest {FunctionId = "FunctionId", Metadata = RpcFunctionMetadata};
+            FunctionLoader.SetupWellKnownPaths(FunctionLoadRequest);
+        }
+
+        internal static PowerShellManager NewTestPowerShellManager(ConsoleLogger logger)
+        {
+            return new PowerShellManager(logger);
```

Comment:
`PowerShellManager` in fact depends on the well known paths to be setup. I think it would be good to have a single way to create an instance of `PowerShellManager`, so that it's guaranteed that the well known paths are setup beforehand (it's done in the static constructor of `TestUtils`).